osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -323,6 +330,8 @@ private void load(OverlayColourProvider colourProvider)
             if (WasDeleted)
                 makeDeleted();
 
+            actionsContainer.AddLink("Copy link", copyUrl);
```

Comment:
The "weird space" you mention is due to this spacing definition on `actionsContainer`:

https://github.com/ppy/osu/blob/47d7d6fad9c677faa32a9e0b11434bdea8762820/osu.Game/Overlays/Comments/DrawableComment.cs#L206

That cannot work like it does currently. Needs some good refactoring.

The cheap way out would be to use `.AddArbitraryDrawable()` and insert a bunch of 10-width spacers between items. The less cheap way would be to make each link a separate drawable and nest those in `actionsContainer`.